### PR TITLE
add armv7 target

### PR DESCRIPTION
**Pull request by KodrAus**
In #41 I ran into some test case failures with ARMv7 on a Raspberry Pi. I'm just opening this up to see if I can replicate those on the targets available to use in CI.

### CODE REVIEW

**Comment by KodrAus** [comment]
Alas it succeeds 🙃

**Comment by thomcc** [comment]
I'm hoping to replicate the CI setup from https://github.com/rust-lang/stdarch/ and/or https://github.com/rust-lang/packed_simd this weekend, which (if successful) will help here. That said looking at what you have there, I think you're running on arm64 which is armv8, no?

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,12 @@ matrix:
       arch: arm64
       env: RUSTFLAGS=-Ctarget-cpu=native
 
+    # Linux (armv7)
+    - name: "armv7-unknown-linux-gnueabihf (neon)"
+      os: linux
+      arch: arm64
+      env: RUSTFLAGS=-Ctarget-feature=+neon
```
Comment (hsivonen) [inline .travis.yml:27]: Adding `+neon` after the fact to the `armv7-unknown-linux-gnueabihf` target results in terrible performance due to the mismatch between the pre-built `std` and what's being built with `+neon`.

The target that doesn't suffer from this problem is `thumbv7neon-unknown-linux-gnueabihf`, which bakes the `neon` option into the target definition and, therefore, comes with `neon`-enabled `std`.

**Comment by KodrAus** [comment]
@thomcc I believe so yes, but am not super familiar with ARM so am deferring to you on it 😄 I was curious to see if I could quickly make it fail the same way in CI for us.

I should have signaled better than this was a quick experiment, and will go ahead and close this.

**Pull request closed without merging**